ebpf-for-windows/ebpf-for-windows.vcxproj

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Why is this file needed in this repo?  It should already be in the ebpf-for-windows project.  Or is this a placeholder until we have a devkit you can install instead of cloning/building the full ebpf-for-windows repo?

Reply:
`ebpf-for-windows` submodule has a solution file. This project is added to perform a custom build step (executes msbuild for `ebpf-for-windows` solution).

I could not find any good way to include a solution file to another solution file. I could have included the individual projects present in `ebpf-for-windows` sln into the top-level solution in this repo, but that will not be scalable.